### PR TITLE
Feat event page

### PR DESCRIPTION
**Pull request by ArshdeepGrover**

### CODE REVIEW

File: projects/commudle-admin/src/app/feature-modules/public-events/components/agenda/event-location-tracks/event-location-tracks.component.ts [hunk 1]
```diff
@@ -18,31 +18,37 @@ export class EventLocationTracksComponent implements OnInit {
   @Input() community: ICommunity;
   @Input() eventLocation: IEventLocation;
   @Output() updateSessionPreference = new EventEmitter();
+  @Output() collapsedChange = new EventEmitter();
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/public-events/components/agenda/event-location-tracks/event-location-tracks.component.ts]: This was removed?

File: projects/commudle-admin/src/app/feature-modules/public-events/components/speakers/speakers.component.ts [hunk 2]
```diff
@@ -9,20 +9,23 @@ import { IUserEventRegistration } from 'projects/shared-models/user_event_regist
 @Component({
   selector: 'app-speakers',
   templateUrl: './speakers.component.html',
-  styleUrls: ['./speakers.component.scss']
+  styleUrls: ['./speakers.component.scss'],
 })
 export class SpeakersComponent implements OnInit {
   @Input() community: ICommunity;
   @Input() event: IEvent;
-  @Output() hasSpeakers = new EventEmitter();
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/public-events/components/speakers/speakers.component.ts:17]: Have you tested for this to work with all types of events?
Without agenda, with agenda, online, offline, etc

File: projects/shared-components/shared-components.module.ts [hunk 3]
```diff
@@ -105,6 +106,7 @@ import { DragDropModule } from '@angular/cdk/drag-drop';
     ShareButtonComponent,
     PaginationComponent,
     LoadingSpinnerComponent,
+    SkeletonLoadingComponent,
```
Comment (arpansac) [inline projects/shared-components/shared-components.module.ts]: There is already a skeleton loader, please use that?

**Review by arpansac — CHANGES_REQUESTED** [review]

File: projects/commudle-admin/src/app/feature-modules/public-events/components/speakers/speakers.component.ts (hunk 2 above)
Comment (ArshdeepGrover) [reply projects/commudle-admin/src/app/feature-modules/public-events/components/speakers/speakers.component.ts:17]: Yes, Tested

**Pull request merged**